### PR TITLE
enable must-gather object refs

### PR DESCRIPTION
**Pull request by gabemontero**
Fixes https://github.com/openshift/cluster-samples-operator/issues/124

/assign @bparees 
/assign @adambkaplan 

@openshift/sig-developer-experience fyi

### CODE REVIEW

File: pkg/operatorstatus/operatorstatus.go [hunk 1]
```diff
@@ -103,6 +103,11 @@ func (o *ClusterOperatorHandler) setOperatorStatus(condtype configv1.ClusterStat
 			state = &configv1.ClusterOperator{}
 			state.Name = ClusterOperatorName
 
+			state.Status.RelatedObjects = []configv1.ObjectReference{
+				{Group: v1.GroupName, Resource: "configs", Name: "cluster"},
+				{Resource: "namespaces", Name: "openshift-cluster-samples-operator"},
```
Comment (bparees) [inline pkg/operatorstatus/operatorstatus.go]: you probably want to gather the openshift namespace too.

Comment (gabemontero) [reply pkg/operatorstatus/operatorstatus.go]: good call

Comment (gabemontero) [reply pkg/operatorstatus/operatorstatus.go]: update pushed @bparees

**Pull request merged**